js-api-doc/value/color.d.ts

```diff
@@ -10,119 +89,467 @@ import {Value} from './index';
  */
 export class SassColor extends Value {
   /**
-   * Creates an RGB color.
+   * Creates an [RGB color].
+   *
+   * If `space` is missing, **only** `undefined` should be passed to indicate a
+   * missing `alpha`. If `null` is passed instead, it will be treated as a
```

Comment:
The term "missing `alpha`" here is a bit confusing, since you're contrasting it with "missing component" below. Maybe "to indicate that `alpha` isn't passed" instead?

Reply:
Addressed in https://github.com/sass/sass/pull/3718/commits/4efd448207eb7e86024a985a89422c87766133ea